crates/holochain/tests/authored_test/mod.rs

```diff
@@ -78,7 +78,7 @@ async fn authored_test() {
         .unwrap();
 
     // Integration should have 3 ops in it.
-    // Plus another 14 for genesis.
+    // Plus another 14 (2x7) for genesis.
```

Comment:
That doesn't make it clearer for me =D Why 2 and why 7?

Reply:
2 agents, and genesis takes 7 ops. Sorry yeah it's not a great deal clearer without context

Reply:
Ah, yes, it's ops, not actions.